packages/doenetml-worker-rust/lib-doenetml-core/src/components/doenet/section.rs

```diff
@@ -35,8 +32,26 @@ mod component {
         )]
         Hidden,
 
+        /// The position of the section relative to other siblings with serial numbers.
+        /// E.g. in `<section /><section />` the first would have serial number 1 and the second 2.
```

Comment:
This description of serial number is incorrect, as is currently 0-indexed (and the tests check that). I think, though, that rather than changing the description, we should actually make `SerialNumber` be 1-indexed to correspond to `CodeNumber`.

Should we also make `SerialNumber` and `CodeNumber` be public? I could see them as being useful for an author to reference. If we do make them public, then `SerialNumber` definitely should be 1-indexed.

Reply:
If we make them public, then we should add a test where they are referenced.

Reply:
I originally had this 1 indexed and then I changed it because: sometimes the labels are not numbers. If we are indexing into an array of `['A', 'B', 'C', ...]` then we want 0-indexed stuff.

I don't think we should expose this right now. I suspect the user will want the displayed serial_number (which could be a letter, etc.) rather than the actual one.

Reply:
Good point. I hadn't thought about non-numbers. Then, I'd just correct the description. Makes sense to come back to the issue of what to expose at a later point when more pieces are in place.